pkg/api/meta/priority.go

```diff
@@ -157,12 +158,19 @@ func (m PriorityRESTMapper) RESTMapping(gk unversioned.GroupKind, versions ...st
 	if err != nil {
 		return nil, err
 	}
-
 	// any versions the user provides take priority
 	priorities := m.KindPriority
 	if len(versions) > 0 {
 		priorities = make([]unversioned.GroupVersionKind, 0, len(m.KindPriority)+len(versions))
 		for _, version := range versions {
+			if strings.Count(version, "/") == 0 {
```

Comment:
This doesn't look right.  Doesn't `ParseGroupVersion` already do this? Also, open an issue to fix this method signature.  Either it gets a version or a groupversion, but we shouldn't have strings floating around anymore.